src/api/Controllers/UploadController.cs

```diff
@@ -258,6 +372,26 @@ private List<BoreholeImport> ReadBoreholesFromCsv(IFormFile file)
         return csv.GetRecords<BoreholeImport>().ToList();
     }
 
+    private List<LithologyImport> ReadLithologiesFromCsv(IFormFile? file)
+    {
+        if (file == null) return new List<LithologyImport>();
+
+        var csvConfig = new CsvConfiguration(new CultureInfo("de-CH"))
+        {
+            Delimiter = ";",
+            IgnoreReferences = true,
+            PrepareHeaderForMatch = args => args.Header.Humanize(LetterCasing.Title),
+            MissingFieldFound = null,
+        };
```

Comment:
PP: Prüfen, ob wir die Config in eine `readonly` Variable auslagern und im Borehole-Import wiederverwenden können.